.github/workflows/ci.yml

```diff
@@ -37,40 +41,43 @@ jobs:
           echo "NAME=${PRETTY_BRANCH_NAME}" >> $GITHUB_OUTPUT
```

Comment:
I can't comment on the line but line 33. Can we bump `actions/checkout` to `v4` to fix the last deprecation?